src/ichmod.cpp

```diff
@@ -144,45 +144,57 @@ main( int argc, char **argv ) {
 void
 usage() {
     char *msgs[] = {
-        "Usage: ichmod [-rhvVM] null|read|write|own userOrGroup dataObj|Collection ...",
+        "Usage: ichmod [-rhvVM] AccessLevel userOrGroup dataObject|Collection ...",
         " or    ichmod [-rhvVM] inherit Collection ...",
         " or    ichmod [-rhvVM] noinherit Collection ...",
-        " -r  recursive - set the access level for all dataObjects",
-        "             in the entered collection and subCollections under it",
+        " ",
+        "Options:",
+        " -r  recursive - set the AccessLevel for all dataObjects",
+        "             in the entered Collection and subCollections under it",
```

Comment:
Consider ...
```
-r  Recursively set the access level for all data objects in the
    collection.
```

Reply:
```
        " -r  recursive - Recursively set the ACCESS_LEVEL for all DataObjects",
        "             in the named Collection and in all subCollections",
```
 ?

Reply:
I don't think having "recursive" after "-r" is helpful. The help text immediately following it describes the option just fine. Also, we can align the next line with the other option's help text.

So, perhaps something like ...
```bash
 -r  Recursively set the ACCESS_LEVEL for all data objects and
     collections in LOGICAL_PATH. LOGICAL_PATH must represent a
     collection.
```

Reply:
done